cli/unipipe/commands/generate.ts

```diff
@@ -87,17 +87,14 @@ function generateUuid() {
 }
 
 async function generateCatalog(options: CatalogOpts) {
-  console.log(catalog);
```

Comment:
I was actually thinking that having these console outputs looks good when you are executing these output commands 😄 it feels like if you agree press enter to write it into a file or cancel the process with cmd+c

Reply:
Personally, I did not like that unipipe cli was spamming command line outputs even if the option `--destination` is specified. I'd expect it to either log or write to a file.

Plus the console output is not useful for scripting anymore if we follow it up to print the interactive part, because piping would also catch the output that asks for user input.